package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-guide",
-  "version": "121.0.1",
+  "version": "122.0.0",
```

Comment:
this is a breaking change - you can't import "COLOR" from "icons/Icon"

Reply:
breaking change = major - so what it's wrong?